nemo_skills/evaluation/metrics/bfcl_metrics.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nemo_skills.evaluation.metrics.base import BaseMetrics
+
+
+class BFCLMetrics(BaseMetrics):
+    """Metrics for BFCL (Berkeley Function Calling Leaderboard) evaluation."""
+
+    # TODO: Ideally we should combine the accuracies across different subsets of the benchmark to
```

Comment:
this is automatically done by summarize_results with a simple average across all subsets. But also if running from eval_group, you can customize this as you already did. I got an idea of how to refactor eval groups and merge them with general eval, so I think after we do this, it will all be supported in a much more natural way